website/client/package-lock.json

```diff
@@ -11676,7 +11676,7 @@
       }
     },
     "habitica-markdown": {
-      "version": "1.4.0",
+      "version": "2.0.0",
       "resolved": "https://registry.npmjs.org/habitica-markdown/-/habitica-markdown-1.4.0.tgz",
```

Comment:
I think you'll have to manually install habitica-markdown@2 with npm i habitica-markdown@2 or the lockfile won't be updated properly

Reply:
Yeah I think it went wrong because the version wasn't available at the time. That situation has improved now! 🥳